README.md

```diff
@@ -69,3 +69,92 @@ The dependencies json lists specific rules for the validator to follow. When lis
 | `valueMatch` | "[field1] must match one of the values from [field2]" | An object in the form [field1]: [field2]. Each value of the object is a field name whose values form the complete list of valid values for the corresponding key, [field1]. The value of [field1] must match one of these values. |
 | `conditions` | An object containing specific conditions that must apply before the appropriate action is taken | An object containing keys such as `length` or `value`, where the value of that key is the length or value that must hold true for the key to be required |
+
+
+## Validator Rules
+In order for a yaml file to be considered "valid", the following conditions must be met:
+### General Formatting
+* The yaml must not contain duplicate keys at the same level
+* The yaml must be in valid yaml format. To check your yaml format, use [yamllint](https://www.yamllint.com/)
+### General Type Checking and Required Keys
+* All value types should follow the `api.yaml` file
+* All keys defined as required by `api.yaml` are required
+* Exceptions to the two rules above include the following:
+    * `scenario.scenes` is required
+    * Each action in `scene.action_mapping` requires `scene_id`
+    * Each action in `scene.action_mapping` requires `choice`
+    * All vitals properties are required
+        * `conscious`
+        * `avpu`
+        * `mental_status`
+        * `breathing`
+        * `hrpmin`
+        * `Spo2`
+    * `injury.status` may only be `hidden`, `discoverable`, or `visible`
+    * `restricted_actions` cannot include `end_scenario` or `tag_character`
+    * `justification` is a prohibited key in `action`
+    * `visited` is a prohibited key in `character`
+    * `session_complete` is a prohibited key in `scenario`
+    * `scenario_complete` is a prohibited key in `state`
+    * `elapsed_time` is a prohibited key in `state`
+    * In `scenes.state`:
+        * Only one `unstructured` property is required in the whole object
+        * `Mission`, `Environment`, `DecisionEnvironment`, and `SimEnvironment` only require the `unstructured` property
+        * `type` is a prohibted key in `SimEnvironment`
+        * `AidDelay` can be empty
+### Dependencies
+#### Conditional Requirements
+* If `scenes[n].action_mapping[m].conditions` has a length of 2 or more, `scenes[n].action_mapping[m].condition_semantics` is required
+* If `scenes[n].transitions` has a length of 2 or more, `scenes[n].transition_semantics` is required 
+* If `state.characters[n].demographics.military_disposition` is “Allied US”, `state.characters[n].demographics.military_branch` is required 
+* If `state.characters[n].injuries[m].name` is “Burn”, `state.characters[n].injuries[m].severity` is required 
```

Comment:
I pretty much always recommend against smart quotes.  They don't add much, and invariably end up getting botched by some downstream system that can't handle them.  Just my opinion; if you want to keep them, I don't have a problem with it.

Reply:
Accidentally used these because I was copying from the word doc into the readme. All fixed now, thanks for catching it!